test/json-stream-test.js

```diff
@@ -0,0 +1,23 @@
+var expect = require('chai').expect;
+var fs = require('fs');
+
+var JSONParseStream = require('../lib/streams/json-parse-stream');
+
+describe('JSONParseStream', function() {
```

Comment:
A test that parses a file containing a single object isn't really testing much.  A better test would be to test a file containing newline-separated JSON objects (see above note).